Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py

```diff
@@ -269,6 +269,52 @@ def test_get_from_version_and_to_version_from_modified_files(self):
         assert filter_envs.get('Demisto one before GA') is False
         assert filter_envs.get('Demisto GA') is False
 
+    def test_get_from_and_to_version_from_modified_files(self):
+        """
+        Given:
+            - Integration with fromversion 6.0.
+            - Playbook with toversion 5.9.9
+            - test playbook with fromversion 5.0.
+            - two_before_ga is '5.0.0'
+            - one_before_ga is '5.1.0'
+            - ga is '5.5.0'
+        When:
+            - running get_test_list_and_content_packs_to_install
+            - running create_filter_envs_file
+        Then:
+            - Create test list with fake_test_playbook
+            - Check that the toversion in the default (99.99.99)
+            - Check minimum version is 5.5.0
+
+        """
+        two_before_ga = '5.0.0'
+        one_before_ga = '5.5.0'
+        ga = '6.0.0'
+        test_path_integration = 'Tests/scripts/infrastructure_tests/tests_data/mock_integrations/' \
+                                'test_fake_integration.yml'
+        test_playbook_path = 'Tests/scripts/infrastructure_tests/tests_data/mock_test_playbooks/' \
+                             'playbook-fake_test_playbook_to_version.yml'
+        playbook_path = 'Tests/scripts/infrastructure_tests/tests_data/mock_playbooks/test_fake_playbook.yml'
+        modified_files_list, modified_tests_list, changed_common, _, sample_tests, modified_metadata_list, _, _ = \
+            create_get_modified_files_ret(modified_files_list=[test_path_integration, playbook_path],
+                                          modified_tests_list=[test_playbook_path])
```

Comment:
Why do you need this in the test?
Why not just create a set of modified files and assert the generated from\to version.

Reply:
Just give different sets of files and see that the from,to version gets the expected values